content/ch-quantum-hardware/Jaynes-Cummings-model.ipynb

```diff
@@ -0,0 +1,764 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gamma=  6.43 kHz\n"
+     ]
+    }
+   ],
+   "source": [
+    "qubit_decay_results = job_qubit_decay.result(timeout=120)\n",
+    "\n",
+    "qubit_decay_values = []\n",
+    "for i in range(len(delay_times)):\n",
+    "    qubit_decay_values.append(qubit_decay_results.get_memory(i)[qubit])\n",
+    "qubit_decay_values = np.abs(qubit_decay_values)\n",
+    "qubit_decay_values = (qubit_decay_values-min(qubit_decay_values))\n",
+    "qubit_decay_values/=max(qubit_decay_values)\n",
+    "\n",
+    "decay_popt,_=fit_exponential(delay_times, qubit_decay_values)\n",
+    "tau=decay_popt[0]\n",
+    "g=1/tau\n",
+    "\n",
+    "plt.scatter(delay_times*1e6, qubit_decay_values, color='black') \n",
+    "plt.plot(delay_times*1e6,exponential(delay_times,*decay_popt),'--',lw=2,color='red',label=r'$\\tau$={:.1f} $\\mu$s'.format(tau*1e6))\n",
+    "plt.title(\"$T_1$ Experiment\", fontsize=15)\n",
+    "plt.xlabel('Delay before measurement [$\\mu$s]', fontsize=15)\n",
+    "plt.ylabel('Signal [a.u.]', fontsize=15)\n",
+    "plt.legend()\n",
+    "plt.show()\n",
+    "\n",
+    "print(r'gamma=  {:.2f} kHz'.format(g/1e3))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### In this chapter we introduce the Jaynes-Cummings model and we consider the relevant quantities for a system consisting of a qubit coupled to a cavity. We use Qiskit Pulse to extract the qubit-resonator coupling strength $g$, the qubit spontaneous emission rate $\\gamma$, and the cavity decay rate $\\kappa$. These parameters combined with the qubit frequency and the resonator frequency that we measured in a previous <a href=\"https://qiskit.org/textbook/ch-quantum-hardware/cQED-JC-SW.html\">chapter</a> allow us to describe our qubit-resonator system."
```

Comment:
Looks like you've left a note in here? Maybe this should be in the introduction.

Reply:
We do also mention this in the introduction. It'd be nice to keep this text in the conclusion